src/librustc_typeck/astconv.rs

```diff
@@ -66,8 +66,9 @@ pub trait AstConv<'gcx, 'tcx> {
     /// late-bound regions.
     fn projected_ty_from_poly_trait_ref(&self,
                                         span: Span,
+                                        item_def_id: DefId,
                                         poly_trait_ref: ty::PolyTraitRef<'tcx>,
-                                        item_name: ast::Name)
+                                        item_name: ast::Name) // FIXME(tschottdorf): remove?
```

Comment:
Looking at the size of this PR, I'd prefer not pulling on this string, unless you have a specific and contained change in mind. We use `item_name` to make a `infer::LateBoundRegionConversionTime::AssocTypeProjection` in this method.